<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectPage</class>
 <widget class="QWidget" name="ConnectPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>0</y>
     <width>891</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame
{
background-color:#1c1d25;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="facelabel">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>14</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="welclabel">
    <property name="geometry">
     <rect>
      <x>683</x>
      <y>13</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
color:#445163;

}</string>
    </property>
    <property name="text">
     <string>Welcome back Vecious</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>891</width>
     <height>641</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame
{
background-color:#242533;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>100</y>
      <width>901</width>
      <height>541</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QListWidget" name="listWidget">
       <property name="styleSheet">
        <string notr="true">QListWidget
{
background-color:#242533;
border:#242533;
}</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QComboBox" name="portcombo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox
{
background-color:#242533;

color:#f0f0f1;
border:#242533;


}
QComboBox::drop-down {border-width: 0px;}
QComboBox::down-arrow 
{image: url(:/resources/img/combo1.PNG);
 padding-right:10px;
}</string>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentText">
     <string>All Servers (30)</string>
    </property>
    <property name="insertPolicy">
     <enum>QComboBox::InsertAtBottom</enum>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="duplicatesEnabled">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>All Servers (30)</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="portcombo_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox
{
background-color:#242533;
color:#979696;
border:#242533;



}
QComboBox::drop-down {border-width: 0px;}
QComboBox::down-arrow 
{image: url(:/resources/img/listicon.PNG);
 padding-right:2px;
}</string>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentText">
     <string>Server Country</string>
    </property>
    <property name="insertPolicy">
     <enum>QComboBox::InsertAtBottom</enum>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="duplicatesEnabled">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>Server Country</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="portcombo_3">
    <property name="geometry">
     <rect>
      <x>176</x>
      <y>60</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox
{
background-color:#242533;
color:#979696;
border:#242533;


}
QComboBox::drop-down {border-width: 0px;}
QComboBox::down-arrow 
{image: url(:/resources/img/listicon.PNG);
 padding-right:2px;
}</string>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentText">
     <string>Server Name</string>
    </property>
    <property name="insertPolicy">
     <enum>QComboBox::InsertAtBottom</enum>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="duplicatesEnabled">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>Server Name</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="portcombo_4">
    <property name="geometry">
     <rect>
      <x>313</x>
      <y>60</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox
{
background-color:#242533;
color:#979696;
border:#242533;


}
QComboBox::drop-down {border-width: 0px;}
QComboBox::down-arrow 
{image: url(:/resources/img/listicon.PNG);
 padding-right:0px;
}</string>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentText">
     <string>Server Encryption</string>
    </property>
    <property name="insertPolicy">
     <enum>QComboBox::InsertAtBottom</enum>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="duplicatesEnabled">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>Server Encryption</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="portcombo_5">
    <property name="geometry">
     <rect>
      <x>542</x>
      <y>65</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox
{
background-color:#242533;
color:#979696;
border:#242533;


}
QComboBox::drop-down {border-width: 0px;}
QComboBox::down-arrow 
{image: url(:/resources/img/listicon.PNG);
 padding-right:7px;
}</string>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentText">
     <string>Server Load</string>
    </property>
    <property name="insertPolicy">
     <enum>QComboBox::InsertAtBottom</enum>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="duplicatesEnabled">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>Server Load</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>211</width>
     <height>701</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame
{
background-color:#1c1d25;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="Logolabel">
    <property name="geometry">
     <rect>
      <x>56</x>
      <y>30</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="connlabel">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>215</y>
      <width>41</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="portlabel">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>270</y>
      <width>41</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="dashlabel">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>327</y>
      <width>41</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>200</y>
      <width>211</width>
      <height>57</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
color:#ffb400;
border:0px solid #000000;
}</string>
    </property>
    <property name="text">
     <string>             CONNECT TO SERVER</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ToPort">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>256</y>
      <width>211</width>
      <height>57</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
color:#445163;
border:0.1em solid #151515;
}</string>
    </property>
    <property name="text">
     <string>PORTFORWARD  </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>312</y>
      <width>211</width>
      <height>57</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
color:#445163;
border:0px solid #000000;
}</string>
    </property>
    <property name="text">
     <string>   VPN DASHBOARD</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
